tests/files/c_source/motohawk_sender.c

```diff
@@ -0,0 +1,131 @@
+/**
```

Comment:
Add a test that generates these (tests/files/c_source/motohawk_sender.h and  tests/files/c_source/motohawk_sender.c) and asserts that the generated code is equal to these files. Right now there is no test of the new functionality.

Reply:
Let me know if tests/files/c_source/motohawk_sender_node.* name is fine or perhaps would be better tests/files/c_source/motohawk_PCM1_node.*.

Reply:
The first one is fine.